crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -368,7 +368,15 @@ impl EventTimelineItem {
 
         match self.content() {
             TimelineItemContent::Message(message) => {
-                matches!(message.msgtype(), MessageType::Text(_) | MessageType::Emote(_))
+                matches!(
+                    message.msgtype(),
+                    MessageType::Text(_)
+                        | MessageType::Emote(_)
+                        | MessageType::Audio(_)
+                        | MessageType::File(_)
+                        | MessageType::Image(_)
+                        | MessageType::Video(_)
+                )
```

Comment:
I see a bit below a function `is_highlighted`. I am curious, are you using it on iOS?

Reply:
I can't find anything - I don't think it has been exposed over the FFI even if we wanted to (but presumably we're happy with the pill colour to highlight a message?).